### PR TITLE
Testing branch merge request

### PR DESCRIPTION
**Pull request by bdowds**
First draft of logging kata
@codytwinton

### CODE REVIEW

File: LoggingKata/Program.cs [hunk 1]
```diff
@@ -1,29 +1,55 @@
 ﻿using System;
 using System.Linq;
 using System.IO;
+using System.Runtime.InteropServices;
```
Comment (codytwinton) [inline LoggingKata/Program.cs]: This using directive is unnecessary.

File: LoggingKata/TacoParser.cs [hunk 2]
```diff
@@ -1,18 +1,64 @@
...
+                return null;
+            }
+
+            if (Math.Abs(longitude) > Point.MaxLon || Math.Abs(latitude) > Point.MaxLat)
+            {
+                _logger.LogWarning("Longitude or Latitude out of bounds");
+                return null;
+            }
+
+            var taco = new TacoBell();
```
Comment (codytwinton) [inline LoggingKata/TacoParser.cs]: You should be using the object initializer.

File: LoggingKata/TacoParser.cs [hunk 3]
```diff
@@ -1,18 +1,64 @@
...
+
+            var cells = line.Split(',');
+
+            if (cells.Length < 2)
+            {
+                _logger.LogWarning("Missing Longitude or Latitude");
+                return null;
+            }
+
+            var longitude = 0.0;
```
Comment (codytwinton) [inline LoggingKata/TacoParser.cs]: You can simplify this file by creating these variables and returning the taco bell in the `try` statement.

**Comment by bdowds** [comment]
@codytwinton completed necessary changes

File: LoggingKata/Program.cs [hunk 4]
```diff
@@ -1,29 +1,54 @@
...
 
             var locations = lines.Select(parser.Parse);
 
-            // TODO:  Find the two TacoBells in Alabama that are the furthurest from one another.
-            // HINT:  You'll need two nested forloops
+            ITrackable location1 = null;
+            ITrackable location2 = null;
+            var distanceMax = 0.0;
+
+            foreach (var LocA in locations)
```
Comment (codytwinton) [inline LoggingKata/Program.cs]: `LocA` and `LocB` should be camelCase: `locA` or `locB`.

**Pull request merged**